strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -30,31 +23,34 @@
 import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 import org.springframework.web.servlet.config.annotation.PathMatchConfigurer;
 import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.xml.bind.Marshaller;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 @Configuration
-@ComponentScan({ "com.carlspring.strongbox.controllers",
-                 "org.carlspring.strongbox.controllers",
-                 "org.carlspring.strongbox.mapper",
-                 "org.carlspring.strongbox.utils",
-                 "org.carlspring.logging" })
-@Import({ CommonConfig.class,
-          StrongboxSecurityConfig.class,
-          StorageApiConfig.class,
-          Maven2LayoutProviderConfig.class,
-          NugetLayoutProviderConfig.class,
-          NpmLayoutProviderConfig.class,
-          StorageCoreConfig.class,
-          SecurityConfig.class,
-          ClientConfig.class,
-          CronTasksConfig.class })
+@ComponentScan({"com.carlspring.strongbox.controllers",
```

Comment:
Could you please, not change the formatting of the code? Such changes that are just changes to the indentation make it hard to spot what exactly changed from all the noise...

Reply:
Fixed